momento-sdk/src/intTest/java/momento/sdk/ListTest.java

```diff
@@ -27,16 +29,18 @@ public class ListTest extends BaseTestClass {
 
   private static final Duration FIVE_SECONDS = Duration.ofSeconds(5);
   private CacheClient target;
-  private String cacheName;
+
+  private final CredentialProvider credentialProvider =
+      new EnvVarCredentialProvider("TEST_AUTH_TOKEN");
+  private final String cacheName = System.getenv("TEST_CACHE_NAME");
```

Comment:
Why not use the EnvVarCredentialProvider for TEST_CACHE_NAME too?

Reply:
Cache name is only relevant for tests (and honestly I'm not sure why we even let the user specify it instead of picking a random one), so we don't need a way to get it in the main code.

Reply:
Ah, I see. 
Yeah, that makes sense if cache name is only being used in tests, we can specify a random name and is not required to be fetched from EnvVarCredentialsProvider. 

Thanks for clarifying!